src/gobupload/relate/table/update_table.py

```diff
@@ -779,9 +778,19 @@ def update(self):
             filename = contents_writer.filename
             confirms = confirms_writer.filename
 
-            for row in result:
-                progress.tick()
-                event = self._create_event(self._format_relation(dict(row)))
-                event_collector.collect(event)
+            try:
+                # Execute the query in a try-except block.
+                # The query is complex and if it fails it is hard to debug
+                # In order to allow debugging, any failing query is reported on stdout
+                # Afterwards the exception is re-raised
+                result = _execute(query, stream=True, max_row_buffer=25000)
```

Comment:
max_row_buffer is a client-side parameter. Given the complex queries only FETCH 100 (sometimes 500) at a time (which takes a long time), I don't think this would do anything for those queries, as the buffer will never reach even close to full. It could help speed up fast queries though. Parameter won't hurt as long as we don't run out of memory (shouldn't be the case with the relate query).